### PR TITLE
[skin.confluence] v4.6.5

### PR DESCRIPTION
**Pull request by DaVukovic**
Updating `skin.confluence` from 4.6.3 to 4.6.5

First time submitting to the skin repo. Hope this is all correct.

### CODE REVIEW

File: skin.confluence/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="skin.confluence" version="4.6.3" name="Confluence" provider-name="Jezz_X, Team Kodi">
+<addon id="skin.confluence" version="4.6.5" name="Confluence" provider-name="Jezz_X, Team Kodi">
```
Comment (enen92) [inline skin.confluence/addon.xml]: remove the website and email tags

Comment (enen92) [reply skin.confluence/addon.xml]: fix also the forum url

Comment (enen92) [reply skin.confluence/addon.xml]: Please drop the comment block in https://github.com/xbmc/repo-skins/blob/d48a1128847d44878a9940dba956d27239a89e55/skin.confluence/language/resource.language.en_gb/strings.po#L19 although this is more of an addon-checker issue

Comment (Rechi) [reply skin.confluence/addon.xml]: The addon-checker is correct, `#` has to be followed by a space (` `), dot (`.`), colon (`:`), comma (`,`) or vertical bar (`|`).

**Comment by DaVukovic** [comment]
meh...I'll fix that later

**Comment by HitcherUK** [comment]
Done, thanks.

https://github.com/xbmc/repo-skins/pull/248

**Pull request merged**